test/e2e/util/util.go

```diff
@@ -115,11 +115,13 @@ func WaitForGuestClient(t *testing.T, ctx context.Context, client crclient.Clien
 
 	t.Logf("Waiting for a successful connection to the guest apiserver")
 	var guestClient crclient.Client
-	waitForGuestClientCtx, cancel := context.WithTimeout(ctx, 5*time.Minute)
+	waitForGuestClientCtx, cancel := context.WithTimeout(ctx, 10*time.Minute)
 	defer cancel()
-	err = wait.PollUntil(5*time.Second, func() (done bool, err error) {
+	// SOA TTL is 60s. If DNS lookup fails on the api-* name, it is unlikely to succeed in less than 60s.
+	err = wait.PollUntil(35*time.Second, func() (done bool, err error) {
 		kubeClient, err := crclient.New(guestConfig, crclient.Options{Scheme: scheme})
 		if err != nil {
+			t.Logf("%s", err)
```

Comment:
Please add some more context to the log rather than just the raw error message, otherwise it will be extremely hard to find this in the log